### PR TITLE
Adds nyc as dev-dependency

### PR DESCRIPTION
**Pull request by msimmer**
This resolves test failures due to the missing nyc package in #9

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,17 +21,18 @@
     "node": ">=4"
   },
   "scripts": {
-    "test": "nyc --reporter=text --reporter=html mocha"
+    "test": "./node_modules/.bin/nyc --reporter=text --reporter=html mocha"
```
Comment (doowb) [inline package.json]: When `nyc` is installed as a `devDependency` this change is not necessary, `npm` automatically includes the `./node_modules/.bin` in the `PATH` when executing.

**Comment by doowb** [comment]
@msimmer thanks for the PR. The tests are failing because of the node versions being tested and what `nyc` supports. I'll get this and the other bug fix merged in and deployed as a patch, then bump the required node version and travis config to node 6 and publish a major version.

/edit: oh... please revert the `test` script line in `package.json` first, then I'll merge. thanks.

File: package.json (hunk 1 above)
Comment (msimmer) [reply package.json]: Got it. I'll fix that up now and push an update now then

Comment (doowb) [reply package.json]: Thanks... I'm going to merge this in, then will you rebase the other PR which will get the tests to run (at least on the good node versions)?

Comment (msimmer) [reply package.json]: Yep, sounds good

Comment (doowb) [reply package.json]: looks like it's not necessary... i was able to restart travis and it pulled in the latests package.json, thanks again!

Comment (msimmer) [reply package.json]: Ah great, thanks!

**Pull request merged**